app/src/main/java/de/danoeh/antennapod/fragment/StatisticsFragment.java

```diff
@@ -0,0 +1,106 @@
+package de.danoeh.antennapod.fragment;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v7.app.AlertDialog;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import java.util.Locale;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.activity.MainActivity;
+import de.danoeh.antennapod.adapter.StatisticsListAdapter;
+import de.danoeh.antennapod.core.storage.DBReader;
+import rx.Observable;
+import rx.Subscription;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.schedulers.Schedulers;
+
+public class StatisticsFragment extends Fragment implements AdapterView.OnItemClickListener {
+
+    public static final String TAG = "StatisticsFragment";
+    private Subscription subscription;
+    private TextView totalTimeTextView;
+    private ListView feedStatisticsList;
+    private StatisticsListAdapter listAdapter;
+
+    //Mandatory Constructor
+    public StatisticsFragment() {
+    }
+
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setRetainInstance(true);
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        totalTimeTextView.setText("...");
+        feedStatisticsList.setVisibility(View.GONE);
```

Comment:
Should set the text and change the visibility in ``loadStats()`` then to make the connection more obvious